Makefile

```diff
@@ -20,9 +20,14 @@ run: ## Run via godep (without building)
 	go run *.go
 
 all: test build docker ## Test, build and docker image build
+	
+fmt: ## Run go fmt for all files in the project (excluding vendor)
+	go fmt $(shell go list ./... | grep -v /vendor/)
```

Comment:
Having this in the make file is good but I think you would improve the development process if you configure your IDE for doing that when a `go` file is saved, because I think you won't want to execute this every time you need to push changes. For example I have in my IDE the commands `gofmt` and [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to apply them when the file is saved.

One thing  we can do for prevent the code is not well formatted is create a [git-hook](https://tip.golang.org/misc/git/pre-commit)